modules/nextflow/src/main/groovy/nextflow/datasource/SraExplorer.groovy

```diff
@@ -43,7 +53,9 @@ import nextflow.util.Duration
 @Slf4j
 class SraExplorer {
 
-    static public Map PARAMS = [apiKey:[String,GString], cache: Boolean, max: Integer, protocol: ['ftp','http','https']]
+    static public Map PARAMS = [apiKey:[String,GString], cache: Boolean, max: Integer, protocol: ['ftp','http','https'],
+                                retryPolicy: Map]
```

Comment:
```suggestion
    static public Map PARAMS = [apiKey:[String,GString], cache: Boolean, max: Integer, protocol: ['ftp','http','https'], retryPolicy: Map]
```

Reply:
Just to know, is there a maximum line length to apply for codestyle?